usmqe_tests/web/others/test_import_cluster.py

```diff
@@ -6,73 +6,33 @@
 
```

Comment:
What is the purpose of test cases here when you just created two modules, one for ceph cluster import, 2nd for the gluster one?